src/Questionnaire/Questionnaire.js

```diff
@@ -0,0 +1,56 @@
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+import { useHistory } from 'react-router-dom'
+import { styleData } from '../data/styleData'
+
+Questionnaire.propTypes = {
+  userStyles: PropTypes.arrayOf(PropTypes.number).isRequired,
```

Comment:
Is the prop "userStyles" used for Styling? If not the wording might mislead.

Reply:
You are right! I renamed it to "userPersonalityStyleIds" to make the function clearer!